### PR TITLE
Update intrinsic signatures.

### PR DESCRIPTION
**Pull request by swatanabe-b1**

### CODE REVIEW

File: esr_key_value_database_intrinsics.md [hunk 1]
```diff
@@ -263,6 +273,11 @@ Returns the new iterator status. It aborts the transaction if `itr` wasn't retur
 If a key is found, the new status is `iterator_ok`. If not found, the new status is
 `iterator_end`. `kv_it_lower_bound` never returns `iterator_erased`.
 
+The size of the key and the size of the value at the iterator's new
+position, or zero if the iterator is at end, will be written to
+found_key_size and found_value_size. If found_key_size overlaps
+found_value_size, aborts the transaction.
```
Comment (arhag) [inline esr_key_value_database_intrinsics.md:279]: Or if they overlap with the range `[key, key + size)`.

File: esr_key_value_database_intrinsics.md [hunk 2]
```diff
@@ -248,6 +253,11 @@ If `itr`'s status is `iterator_end` then this finds the last non-deleted key
 in range. If found, the new status is `iterator_ok`. If not found, the new
 status is `iterator_end`.
 
+The size of the key and the size of the value at the iterator's new
```
Comment (tbfleming) [inline esr_key_value_database_intrinsics.md:256]: Need to update prototype above

File: esr_key_value_database_intrinsics.md (hunk 1 above)
Comment (tbfleming) [reply esr_key_value_database_intrinsics.md:279]: Need to update prototype above

**Comment by swatanabe-b1** [comment]
Oops.  Pushed the wrong branch.

**Pull request merged**